src/chocolatey/infrastructure.app/nuget/NugetCommon.cs

```diff
@@ -58,6 +58,7 @@ public static IPackageRepository GetRemoteRepository(ChocolateyConfiguration con
 
             // ensure credentials can be grabbed from configuration
             HttpClient.DefaultCredentialProvider = new ChocolateyNugetCredentialProvider(configuration);
+            HttpClient.DefaultCertificateProvider = new ChocolateyClientCertificateProvider(configuration);

```

Comment:
I totally missed that these are different properties. Nevermind, nothing to see here.